shared/chat/conversation/index.desktop.js

```diff
@@ -145,7 +145,7 @@ export default branch(
   (props: Props) => props.selectedConversation === Constants.nothingSelected,
   renderComponent(NoConversation),
   branch(
-    (props: Props) => !!props.rekeyInfo,
+    (props: Props) => !!props.rekeyInfo && !props.finalizeInfo,
```

Comment:
if we need to rekey and yet its finalized there's nothing we can do currently. kbfs won't rekey for finalized folders by design